presto-native-execution/src/test/java/com/facebook/presto/spark/TestPrestoSparkNativeExecution.java

```diff
@@ -47,34 +46,39 @@
  * requests. This port number has to be the same as to which your externally launched process listens.
  * Example: -DNATIVE_PORT=7777.
  * When NATIVE_PORT is specified, PRESTO_SERVER argument is not requires and is ignored if specified.
+ * <p>
+ * For test queries requiring shuffle, the disk-based local shuffle will be used. For more details of the local shuffle,
+ * Please refer to:
+ * <a href="https://github.com/prestodb/presto/blob/master/presto-native-execution/presto_cpp/main/operators/LocalPersistentShuffle.h">Local Shuffle</a>
```

Comment:
Let's drop 'https://github.com/prestodb/presto/blob/master/presto-native-execution' as this file is in the same repo.